### ISSUE TITLE
Build custom docker image

### ISSUE THREAD

**Issue by chrgraefe** [Nordes/IdentityServer4.LdapExtension#49]
I would like to build a "standard" docker image for IdentityServer4.LdapExtension.

This would ease the entry barrier to participate in development. Potential develop doesn't have to configure there own docker image.

any thoughts?

**Comment by Nordes** [comment]
Yes, it's not something too difficult to do. But you might have to have a samba/ldap/similar on the same image which would not be "standard" for docker image which you should have only 1 process running. That being said, it's feasible.

For example, if you have a docker compose, you could have Redis+LDAP+IS4 running and then you could tests. 

(I will come back eventually to work on my projects, i've been quite busy in the last months)
